test/TestDataFactory.js

```diff
@@ -0,0 +1,36 @@
+// ## TestDataFactory functions
+// Emulate old, pre-RDF/JS output (plain strings instead of Terms) to make tests pass
+
+// ### Creates an IRI
+function namedNode(iri) {
+  return iri;
+}
+
+// ### Creates a blank node
+function blankNode(name) {
+  return '_:b' + name;
+}
+
+// ### Creates a literal
+function literal(value, tag) {
+  if (tag) {
+    if (tag.length < 10) // hacky "heuristic" for test purposes only
+      return value + '@' + tag; // language tag
+    return value + '^^' + tag; // typed literal
```

Comment:
I did one more update and removed the quoting from typed literals here.

The failures went up to 38, but all of them involve `xsd:integer` values, so I think it's `SparqlGenerator` treating integers specially, and there is some symmetry missing.